horace_core/sqw/PixelData/@PixelDataFileBacked/private/set_prop_.m

```diff
@@ -20,10 +20,5 @@
     obj.is_misaligned_ = false;
     obj.alignment_matr_= eye(3);
 end
-
-% Single call to this setter will lead to invalid results as to be
-% correct, the check should run over whole pages range.
 %
-% it will be correct if set_prop was run within the loop over
-% whole file and initial range was initialized properly
-obj=obj.reset_changed_coord_range(fld);
+obj = obj.invalidate_range(fld);
```

Comment:
could you reinstate with corrections the useful comments at old lines 24-27

Reply:
The rows 27-29 are not valid any more. I am not recalculateing range here, so the comments make no sence.